src/lang/ADempiere/es.js

```diff
@@ -531,6 +531,16 @@ export default {
       toolsPoint: {
         title: 'Herramientas del Punto de Venta'
       }
+    },
+    charge: {
+      title: 'Crear cuenta y cargo',
```

Comment:
I think that for the translations we should separate the files by translation module, since the file `/src/lang/ADempiere/es.js` or `/src/lang/ADempiere/en.js`, are becoming large and unmaintainable, there are translations that are never used, could be for example `forms.js` or `pos.js` where only the translations are referenced and make the import and export of `/src/lang/ADempiere/index.js`.

This could be a change in another pull request.

Reply:
Hardcode here, very bad guy